src/content/projects/vvd-viewer.md

```diff
@@ -11,21 +11,14 @@ documentation link: https://github.com/JaneliaSciComp/VVDViewer/wiki
 installation instructions link: https://github.com/JaneliaSciComp/VVDViewer?tab=readme-ov-file#building-vvdviewer
 how to cite link: https://doi.org/10.5281/zenodo.5841616
 how to cite text: https://doi.org/10.5281/zenodo.5841616
-# additional links array:
-#   [
-#     Optional additional external links,
-#     For example - to a project hompage,
-#   ]
-# additional links text array:
-#   [
-#     The display text you wish to appear for each link provided above,
-#     Display text for link 2,
-#   ]
+additional links array:
+  [https://github.com/JaneliaSciComp/VVDViewer/releases/latest]
```

Comment:
Note - Although it appears deleted in the above comment, the link to the binaries is still in the file!